README.md

```diff
@@ -1 +1,107 @@
+# Introduction to UCM Starlight
+
 ![ucm arch](https://github.com/raycarroll/ucm/assets/3717348/2f0870d1-b90d-4a3d-b2d2-6fd8440d0959)
+
+## Setup guide
+
+### Prerequisites:
+
+- oc https://docs.openshift.com/container-platform/4.8/cli_reference/openshift_cli/getting-started-cli.html
+- OpenShift cluster and it's admin credentials for further actions 
```

Comment:
We should stick with native Kubernetes commands rather than require OpenShift explicitly as they have already expressed concern about access to OpenShift, and there is a lack of clarity/understanding on the differences.

Reply:
Also, we should probably add specific steps for making change, i.e. building/pushing the image and then flag that changes need to be made to the deployment files